api/api/specs/experiment-engines.yaml

```diff
@@ -296,6 +296,12 @@ components:
           $ref: "#/components/schemas/ExperimentEngineType"
         config:
           type: "object"
+        plugin_config:
+          type: "object"
+          nullable: true
+          properties:
+            image:
+              type: "string"
```

Comment:
Updated OpenAPI specs and added support for `plugin_config` returned as part of `ExperimentConfig` by Turing, as implemented in #158.

Reply:
I'm not sure if this needs to be exposed to the users. But for the very least, it should be `read-only`, as it creates a security concern (i.e. Turing users can specify an arbitrary plugin, that will be executed inside the cluster and cause damage. Plugins configuration should be in a full control of Turing operators, ie. those who are deploying Turing).

Reply:
Okay, I'll not specify the schema of `plugin_config` then, and leave it as a arbitrary `read_only` object.

Reply:
Actually, since it's going to be `read-only`, I think I'd just specify the spec for `ExperimentConfig` as follows to remove any mention of `plugin_config` in the OpenAPI spec, as well as in the SDK:

```yaml
ExperimentConfig:
  type: "object"
  required:
    - type
  properties:
    type:
      $ref: "#/components/schemas/ExperimentEngineType"
    config:
      type: "object"
  additionalProperties:
    type: "object"
    readOnly: true
    nullable: true
```